### ISSUE TITLE
Issues tab

### ISSUE THREAD

**Issue by sjehuda** [OpenUserJS/OpenUserJS.org#2031]
### What is missing?

On Profile bar there are the following links, and issues would be beneficial to have:

- Scripts
- **Issues**
- Syncs
- Comments
- Votes
- Flags

An issues page where all issue are viewable, just like comments would be helpful.

### Why?

1) One might be aware that there are open issues with some userscript bu would not recall how many, so attention is neglected.
2) The current notification is orange comment bubble which can only be seen per page, which would mak it tedious to browse page by page.

**Comment by sjehuda** [comment]
Another benefot would be this:

When there are flagged scripts, the issue tab would turn into red or have a red underline or another indication of sort.

**Comment by Martii** [comment]
Thanks for the issue.

While this is possible it is not feasible. Multiple calls to the DB for someone who has hundreds of scripts would lag the system.

So no. This was similarly discussed with other areas about a decade ago... same conclusions.

**Comment by sjehuda** [comment]
What about a message at top?

> "Last comment from _Mr. Userscript_ at _Timestamp_ on _Your Userscript_.".

And also a message about an urgent problem.

> "WARNING! Your account is facing a deletion due to a violation of the terms of OUJS. You are required to cooperate with the OUJS Admins at _This Issue_..

The only reasons (or factors) I noticed that I have had a problem were:
1) I was asking a question about two userscripts with the same purpose.
2) Then, while posting the question, I noticed that a userscript was hidden.
2) After checking that the userscript still exist (hidden), I noticed the flag and the issue.

I think had no other mean to notice this.
